2.0/Box/glTF-pbrSpecularGlossiness/Box.gltf

```diff
@@ -112,8 +112,26 @@
                     0.0,
                     0.0,
                     1.0
-                ]
-            }
+                ],
+                "metallicFactor": 0.0
+            },
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [
+                        0.800000011920929,
+                        0.0,
+                        0.0,
+                        1.0
+                    ],
+                    "specularFactor": [
+                        0.20000000298023225,
```

Comment:
@bghgary 
Btw, should such case (both models defined for the same material, `specularFactor` isn't 0.04) be a validation warning?